Architecture/TodoApp/android/core/src/main/java/com/novoda/todoapp/tasks/presenter/TasksPresenter.java

```diff
@@ -28,15 +30,17 @@
     private final TasksService tasksService;
     private final TasksLoadingDisplayer loadingDisplayer;
     private final TasksDisplayer tasksDisplayer;
+    private final NavDrawerDisplayer navDrawerDisplayer;
     private final Navigator navigator;
 
     private CompositeSubscription subscriptions = new CompositeSubscription();
     private TasksActionListener.Filter currentFilter = TasksActionListener.Filter.ALL;
 
-    public TasksPresenter(TasksService tasksService, TasksDisplayer tasksDisplayer, TasksLoadingDisplayer loadingDisplayer, Navigator navigator) {
+    public TasksPresenter(TasksService tasksService, TasksDisplayer tasksDisplayer, TasksLoadingDisplayer loadingDisplayer, NavDrawerDisplayer navDrawerDisplayer, Navigator navigator) {
```

Comment:
Fixed in https://github.com/novoda/spikes/pull/138/commits/ed8393e2673e55d5df2079d36aa37e7375044de2